pages/records/download-va-letters/letters.md

```diff
@@ -1,9 +0,0 @@
----
-title: Download VA Letters and Documents
-entryname: letters
-layout: page-react.html
-includeBreadcrumbs: true
-in_maintenance: false
-maintenance_line1: We’re sorry. The page you’re looking for is currently down while we fix a few things. We’ll get it back online as soon as we can.
-maintenance_line2: In the meantime, please try the search box above or one of the options listed below to find more information.
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-b8b1c4dd453cb095be9e305eb8a2665fR5.